go/cmd/yolo/main.go

```diff
@@ -276,6 +277,43 @@ func yolo(args []string) error {
 		},
 	}
 
+	tree := &ffcli.Command{
+		Name:    `tree`,
+		FlagSet: storeFlagSet,
+		Options: []ff.Option{ff.WithEnvVarNoPrefix()},
+		Exec: func(_ context.Context, _ []string) error {
+			logger, err := loggerFromArgs(verbose, logFormat)
+			if err != nil {
+				return err
+			}
+			db, err := dbFromArgs(dbStorePath, logger)
+			if err != nil {
+				return err
+			}
+			defer db.Close()
+
+			svc, err := yolosvc.NewService(db, yolosvc.ServiceOpts{
+				Logger:  logger,
+				DevMode: true,
+			})
+			if err != nil {
+				return err
+			}
+
+			ctx := context.Background()
+			input := &yolopb.DevDumpObjects_Request{
+				WithPreloading: withPreloading,
```

Comment:
```suggestion
				WithPreloading: true,
```

`tree` without this makes no sense I guess.